src/main/java/com/squareup/kotlinpoet/TypeSpec.kt

```diff
@@ -452,12 +457,13 @@ class TypeSpec private constructor(builder: TypeSpec.Builder) {
     }
 
     fun addProperty(propertySpec: PropertySpec) = apply {
-      if (kind == Kind.EXPECT_CLASS) {
+      if (kind == Kind.EXPECT_CLASS || kind == Kind.EXTERNAL_CLASS) {
+        val modifier = if (kind == Kind.EXPECT_CLASS) "expect" else "external"
         require(propertySpec.initializer == null) {
-          "properties in expect classes can't have initializers"
+          "properties in $modifier classes can't have initializers"
```

Comment:
I think it's fine to just print `$kind` here

Reply:
done